src/DiffEqSensitivity.jl

```diff
@@ -30,7 +30,7 @@ export extract_local_sensitivities
 export ODEForwardSensitivityFunction, ODEForwardSensitivityProblem, SensitivityFunction,
        ODEAdjointSensitivityProblem, ODEAdjointProblem, AdjointSensitivityIntegrand,
        adjoint_sensitivities, adjoint_sensitivities_u0, Sobol, Morris, gsa,
-       SensitivityAlg, regression_sensitivity, DGSM, eFAST
+       SensitivityAlg, Regression, DGSM, eFAST
```

Comment:
That's probably too common of a name to export.

Reply:
```suggestion
       SensitivityAlg, RegressionGSA, DGSM, eFAST
```
?